vision/classification/leafsnap30/README.md

```diff
@@ -0,0 +1,69 @@
+# LeafSnap30
+
+## Description
+LeafSnap30 is a Neural Network model trained on the [LeafSnap 30 dataset](https://zenodo.org/record/5061353/). It addresses an image classificaiton task- identifying 30 tree species from images of their leaves. This task has been approached with classical computer vision methods a decade ago on more species dataset containing artifacts, while this model is trained on a smaller and cleaner dataset, particularly useful for demonstrating NN classification on simple, yet realistic scientific task.
+
+## Model
+
+|Model        |Download  | Download (with sample test data)|ONNX version|Opset version|Accuracy |
+|-------------|:--------------|:--------------|:--------------|:--------------|:--------------|
+|Model Name       | Relative link to ONNX Model with size  | tar file containing ONNX model and synthetic test data (in .pb format)|ONNX version used for conversion| Opset version used for conversion|Accuracy values |
+|LeafSnap30|    [1.48 MB](model/leafsnap_model.onnx)    | [1.55 MB](model/leafsnap30.tar.gz) | 1.9.0  |11 | train: 95%, validation: 86, test: 83%     |
```

Comment:
I would suggest using leafsnap-11.onnx as the model name (to represent the used opset_version) and please use consistent name for the .tar.gz file like leafsnap-11.tar.gz.